src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/itemroutermodules/DirectionalItemRouterModule.java

```diff
@@ -227,17 +227,19 @@ protected String[] makeDirectionalLore(String... lore) {
     }
 
     @Override
-    public boolean onSlotClick(HumanEntity player, int slot, ClickType click, ItemStack inSlot, ItemStack onCursor) {
```

Comment:
Instead of removing the onSlotClick() function entirely, I tried changing the function to:
```
    public boolean onSlotClick(HumanEntity player, int slot, ClickType click, ItemStack inSlot, ItemStack onCursor) {
        if (onCursor.getType() == Material.AIR) {
            gui.getInventory().setItem(slot, null);
        } else {
            ItemStack stack = onCursor.clone();
            stack.setAmount(onCursor.getAmount());
            onCursor.setAmount(0);
            gui.getInventory().setItem(slot, stack);
        }
        return false;
    }

Reply:
This makes it so it takes whatever you have on your cursor and puts it in the filter slot. My problem with my fix is once that's done, you cannot retrieve that item back. Not a good solution imo, but better than a dupe?

Reply:
Do you know how the dupe occurs with this? I'd rather fix the issue than remove the feature :D

Reply:
Double clicking items inside your inv to "pull" all items of the same type from opened containers onto your mouse. Since the item in the filter is cloned, the double click will pull that item and add it to your cursor stack.

Reply:
Gotcha, where is (iirc) a ClickType called Pull-to-cursor (or something in that vein) that is likely the event that wants to be stopped here. The other way, which I think I used in a previous fix for the BSU dupe, is too apply a custom lore/pdc/whatever element to the item so they no longer stack with the original item and can no longer be drawn to the cursor.

Reply:
so which one i should make?

Reply:
Whichever fixes the issue. You just need to stop the pull to cursor action via whatever means is most suitable. Removing the feature as you first describe here is not suitable :)

Reply:
wouldnt adding a nbt tag/lore make filter check for wrong item?

Reply:
Likely, you would have to compensate in some manner for that. Check out my PR here: https://github.com/Slimefun/SensibleToolbox/pull/85/files which was for virtually the same issue. I had to split out `stored` (item item used for comparison) and then added `storedDisplay` which is what was used to display the item back to the user whilst removing the dupe. You may not be able to apply it directly, but it's there and a working solution.